vmdb/app/apis/miqservices_ops.rb

```diff
@@ -244,6 +243,7 @@ def agent_config(hostId, config)
 
   def agent_register(xmlFile)
     ret = false
+    require 'miq-xml'
```

Comment:
@Fryguy yeah, so I like files stating their dependencies.  We have too many files that don't state their needs and the moment you pull them apart, it blows up in unexpected ways.  I'm fine with putting `autoload 'MiqXml', 'miq-xml'` at the top of the files that use MiqXml instead of burying the require in each method.